NUnit3TestAdapter.sln

```diff
@@ -4,14 +4,12 @@ VisualStudioVersion = 16.0.28606.126
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{7CE30108-5D81-4850-BE6B-C8BCA35D3592}"
 	ProjectSection(SolutionItems) = preProject
-		acceptance.cake = acceptance.cake

```

Comment:
Did you remove the acceptance parts ?

Reply:
`acceptance.cake` was removed a while back when I added the acceptance test csproj. This should have been removed in that PR.